main.nf

```diff
@@ -11,27 +11,11 @@
 
 nextflow.preview.dsl = 2
 
-/*
- * Print help message if required
- */
-if (params.help) {
-    // TODO nf-core: Update typical command used to run pipeline
-    def command = "nextflow run nf-core/pangenome --input samplesheet.csv -profile docker"
-    log.info Headers.nf_core(workflow, params.monochrome_logs)
-    log.info Schema.params_help("$baseDir/nextflow_schema.json", command)
-    exit 0
-}
-
-/*
-* TODO integrate this later....
-*/
-
-/*
 params.input_fasta="${baseDir}/**.fa.gz"
-params.map_pct_id=false
-params.align_pct_id=false
-params.n_secondary=false
-params.segment_length=false
+params.map_pct_id=90//false
```

Comment:
Should eventually move to nextflow.config, but fine if we ignore it here

I can do that tomorrow maybe

Reply:
Yes, as stated above, this is still a TODO.